sourcemash/frontend/static/js/routers/app_router.js

```diff
@@ -6,6 +6,7 @@ Sourcemash.Routers.AppRouter = Backbone.Router.extend({
         "categories/:id": "showCategory",
         "saved": "showSaved",
         "browse": "browseFeeds",
+        "categorizer": "showCategorizer",
```

Comment:
No where else

Alex Gerstein
gerstein.alex@gmail.com




On Sun, May 24, 2015 at 7:56 PM -0700, "Scott Gladstone" <notifications@github.com> wrote:












In sourcemash/frontend/static/js/routers/app_router.js:
> @@ -6,6 +6,7 @@ Sourcemash.Routers.AppRouter = Backbone.Router.extend({
>          "categories/:id": "showCategory",
>          "saved": "showSaved",
>          "browse": "browseFeeds",
> +        "categorizer": "showCategorizer",

So where besides the splash page are you saying to put this?



—
Reply to this email directly or view it on GitHub.